src/index.js

```diff
@@ -1,7 +1,31 @@
-import { sanitizeInputCommand } from './utils/helper.js';
+import { sanitizeInputCommand , readAndParseTomlConfig} from './utils/helper.js';
 import fs from 'fs';
 let args = process.argv.slice(2);
 
+// Check for config flag
+const configFlagIndex = args.findIndex(arg => arg === '-c' || arg === '--config');
+
+//These lines check if the user has given the `-c` or `--config` flag followed by a file path.
+if (configFlagIndex !== -1 && args[configFlagIndex + 1]) {
+  //If yes, they save that path in `configFile`.
+    const configFile = args[configFlagIndex + 1];
+    
+    if (!fs.existsSync(configFile)) {
+        console.error('Configuration file not found:', configFile);
+        process.exit(-1);
+    }
+    
+    const config = readAndParseTomlConfig(configFile);
```

Comment:
When a user passes a version or help flag within toml config file such as 

```bash
version: true
.. all other flags
```
or 
```bash
... all other flags
version: true
```
or
```bash
.. all other flags
version: true

```

Reply:
Resolved in commit [8371c9eb](https://github.com/sdthaker/Node-TILify/pull/13/commits/21441eaecd095e242fb3dc277e10194bf512c3e2)

I tried to do duplicate your logic to print out the name and version but didn't work, currently name and version are hard coded.

Reply:
Yes, thanks for letting me know! I'll be fixing that in the next lab.